pbspro.spec

```diff
@@ -513,7 +513,6 @@ fi
 %exclude %{pbs_prefix}/lib*/python/pbs_bootcheck*
 %exclude %{pbs_prefix}/libexec/install_db
 %exclude %{pbs_prefix}/libexec/pbs_habitat
-%exclude %{pbs_prefix}/libexec/pbs_schema_upgrade
```

Comment:
why did you remove this?

Reply:
This change was added only in pbspro.spec file as part of 1f8828f022bfd7e08cf3a53c55902c455ad8d74d, I have just synced with the latest changes by running autogen.sh. 

The right fix would be adding the change in the proper file "pbspro.spec.in". I have added it now.

Thanks for your comment, I have pushed it, plz review.